api/views/purchaseimport.py

```diff
@@ -26,23 +28,50 @@ def __init__(self, **kwargs):
         self.errors = []
         self.start = None
         self.file_digest = None
+        self.tmp_id = uuid.uuid4()
         self.file = None
         super().__init__(**kwargs)
 
+    def _process_file(self):
+        file_hash = hashlib.md5()
+        chunk = []
+        batch_i = 1
```

Comment:
Je trouves le nom de ce variable pas clair - c'est pour suivre le nombre de lignes lit par le loop ?
Dans ce cas je dirais `row_count` c'est plus explicit (sinon, batch_i je pense que c'est combien de chunks ont été traités)

Reply:
Done in 111d249c132b67c660fa8f7f4ba4ad3688e0160c